index.js

```diff
@@ -0,0 +1,2 @@
+require('./dist/json-formatter');
+module.exports = 'jsonFormatter';
```

Comment:
Oh got it. Makes sense but this might break things for people who are using `main` for injecting a scripts in their HTML without webpack. 

To support both cases, we can export the string in `dist/json-formatter.js` after we checked export exists. Then you can do `angular.module('myApp', [require('jsonformatter')]);` or am I missing something?